code/src/reducers/todos.js

```diff
@@ -0,0 +1,37 @@
+import {  createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  list: {
+    name: 'start',
+    items: [
+      {
+        description: '',
+        done: false
+      }
+    ]
+  }
+};
+
+export const todos = createSlice({
+ name: 'todos',
+ initialState: initialState,
+ reducers: {
+   addTodo: (state, action) => {
+     const itemInfo = action.payload;
+     state.list.items.push(itemInfo)
+   },
+   setDone: (state, action) => {
+     const { itemIndex, done } = action.payload;
```

Comment:
It might be that it is a late friday night when I'm doing this review, but I don't 100% get this, maybe you can explain it to me when you've read this :)